content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -0,0 +1,186 @@
+
+We also recommend using environment variables when possible to avoid unnecessary credential disclosure. Terraform runs receive the full text of [sensitive](#sensitive-values) Terraform variables and may print the value in logs and state files if the configuration sends the value to an output or a resource parameter. Sentinel mocks downloaded from runs will also contain the sensitive values of Terraform variables. Environment variables can also be included in log files if `TF_LOG` is set to `TRACE`.
+
+### Variable Limits
+The following limits apply to variables:
+
+Component   |  Limit
+------------|---------------------------
+description |  512 characters
+key         |  128 characters
+value       |  256 kilobytes
+
+### Multi-Line Text
+
+You can type or past multi-line text into variable value text fields.
+
+### HashiCorp Configuration Language (HCL)
+
+You can use HCL for Terraform variables, but not for environment variables. The same Terraform version that performs runs in the workspace will interpret the HCL.
+
+Variable values are strings by default. To enter list or map values, click the variable’s **HCL** checkbox (visible when editing) and enter the value with the same HCL syntax you would use when writing Terraform code. For example:
+
+```hcl
+{
+    us-east-1 = "image-1234"
+    us-west-2 = "image-4567"
+}
+```
+### Sensitive Values
+
+~> **Warning:** There are some cases when even sensitive variables are included in logs and state files. Refer to [security](#security) for more information.
+
+Terraform often needs cloud provider credentials and other sensitive information that shouldn't be widely available within your organization. To protect these secrets, you can mark any Terraform or environment variable as sensitive data by clicking its **Sensitive** checkbox that is visible during editing.
+
+Marking a variable as sensitive prevents all users (including you) from viewing its value in the variables section of the workspace in Terraform Cloud's UI or through the Variables API endpoint.
+
+Users with permission to read and write variables can set new values for sensitive variables, but no other attribute of a sensitive variable can be modified. To update other attributes, delete the variable and create a new variable to replace it.
+
+[permissions-citation]: #intentionally-unused---keep-for-maintainers
+
+
+### Variable Description
+
+~> **Warning:** Variable descriptions are not encrypted, so do not include any sensitive information.
+
+Optional variable descriptions help distinguish between similarly named variables. They are only shown on the **Variables** page and are completely independent from any variable descriptions declared in Terraform CLI.
```

Comment:
Specify that the descriptions themselves are optional. Old wording could also mean that we're only referring to the descriptions of optional variables. 

```suggestion
Variable descriptions are optional, and help distinguish between similarly named variables. They are only shown on the **Variables** page and are completely independent from any variable descriptions declared in Terraform CLI.
```